templates/CRM/Contact/Form/Contact.tpl

```diff
@@ -23,10 +23,10 @@
     {include file="CRM/common/formButtons.tpl" location="top"}
     </div>
 
-    <div class="crm-accordion-wrapper crm-contactDetails-accordion">
```

Comment:
The JS from line 92 onwards interacts with the accordion class names here (called 'tabs' in the comments): e.g. hides them where there's no fields, opens if there's an error, and opens/closes all with the 'expand all tabs'/'collapse all tabs' link top right. This PR breaks that - so either the JS needs removing or rewriting (ref @colemanw). I guess this might depend on how far away the Form Builder version is.

Reply:
I noticed the same issue. The JS could just target all details on the page, assuming there are no details that should *not* be toggled.

Reply:
@vingle this is what I changed in this file to get the expand/collapse all tabs to work:

```
$('a#expand').click( function() {
      if( $(this).attr('href') == '#expand') {
        var message = {/literal}"{ts escape='js'}Collapse all tabs{/ts}"{literal};
        $(this).attr('href', '#collapse');
        $('.crm-form-block details:not([open])').crmAccordionToggle();
      }
      else {
        var message = {/literal}"{ts escape='js'}Expand all tabs{/ts}"{literal};
        $('.crm-form-block details[open]').crmAccordionToggle();
        $(this).attr('href', '#expand');
      }
      $(this).html(message);
      return false;
    });
```

Reply:
That looks good to me. I guess at some point we could add details about crmAccordionToggle to the docs.

Reply:
@aydun do you prefer I make a PR against your branch, or is including a code snippet enough?

Reply:
@herbdool - probably best if you do a PR against mine - thanks.

Reply:
@aydun here's the PR https://github.com/aydun/civicrm-core/pull/1